includes/amp-helper-functions.php

```diff
@@ -917,6 +917,10 @@ function amp_get_content_sanitizers( $post = null ) {
 		'AMP_Style_Sanitizer'             => [
 			'include_manifest_comment' => ( defined( 'WP_DEBUG' ) && WP_DEBUG ) ? 'always' : 'when_excessive',
 		],
+		'AMP_Link_Sanitizer'              => [
+			'add_query_vars'    => ! amp_is_canonical(),
```

Comment:
We'll want to only enable AMP-to-AMP by default in Transitional mode, since in Reader mode the AMP version is not full parity with the non-AMP version. So by default users should navigate to non-AMP pages when leaving Reader mode templates.

```suggestion
			'add_query_vars'    => AMP_Theme_Support::TRANSITIONAL_MODE_SLUG === AMP_Theme_Support::get_support_mode(),
```